lntest/node.go

```diff
@@ -938,7 +938,8 @@ func (hn *HarnessNode) WaitForBalance(expectedBalance int64, confirmed bool) err
 	err := WaitPredicate(doesBalanceMatch, 30*time.Second)
 	if err != nil {
 		return fmt.Errorf("balances not synced after deadline: "+
-			"expected %v, only have %v", expectedBalance, lastBalance)
+			"expected %v, only have %v",
+			btcutil.Amount(expectedBalance), lastBalance)
```

Comment:
Alternative could be to change the func argument to `btcutil.Amount`.

Reply:
`fmt.Errorf` call not formatted according to guide lines.

Reply:
> Alternative could be to change the func argument to btcutil.Amount.

Yeah, I would also prefer that. Turned out we are using the proto fields (which are `int64`) directly many places, so the diff became too big to make it worth it I think.

> fmt.Errorf call not formatted according to guide lines.

We usually try to keep logs and errors over as few lines as possible.

Reply:
Actually, the diff didn't become that bad, so I'll do what you initially suggested :)

Reply:
>We usually try to keep logs and errors over as few lines as possible.

Ah, I see now that that exception indeed landed in the contribution guidelines.